src/common/utils.h

```diff
@@ -33,14 +33,12 @@
 #include <pthread.h>
 #include <random>
 #include <sstream>
-#include <stdlib.h>
 #include <string>
 #include <thread>
-#include <time.h>
 #include <unistd.h>
 #include <vector>
 
-#define MAX_CORE_NUM 200
+#define MAX_CORE_NUM (200)
```

Comment:
Why do we need the brackets here?

Reply:
It is safest (relatively) to always put parentheses around macros since they are "text substitution".  It sometimes helps with unexpected expansions when combining or in specific uses.  Eventually, we can remove all the macros since they scare little children.